src/pages/login/login.scss

```diff
@@ -5,6 +5,8 @@ page-login {
     flex-direction: column;
     justify-content: flex-start;
     align-items: center;
+    height: 100vh;
```

Comment:
careful about using vh, is this tested on mobile sizes?

Reply:
For the most part, I have not touched the css or tested the mobile version. Only desktop Web at the moment

Reply:
The mobile versions are built with the same code base - you can test for the mobile sizes using device toolbar in your browser. You've added a height of 100vh here, is it an accident or is it needed?

Reply:
I did not add it...

Reply:
I see what happened - nonetheless, these changes that are non-related to your feature is being introduced in your PR. Seems like you branched off from a starting point that included Dom's changes here: https://github.com/EktaFlow/docent-base/pull/178/commits/3b3bd2e90b3bb4462992f3c28f7bacad030ae1b8

I'd recommend parsing through your commits and cherry picking only the commits related to your PR.